Cargo.toml

```diff
@@ -207,7 +207,7 @@ procinfo = { git = "https://github.com/tikv/procinfo-rs", rev = "6599eb9dca74229
 # kvproto at the same time.
 # After the PR to kvproto is merged, remember to comment this out and run `cargo update -p kvproto`.
 # [patch.'https://github.com/pingcap/kvproto']
-# kvproto = {git = "https://github.com/your_github_id/kvproto", branch="your_branch"}
+# kvproto = { git = "https://github.com/your_github_id/kvproto", branch="your_branch" }
```

Comment:
White spaces could be removed.

Reply:
It seems that when I use these patches here, if I do not add these spaces, TOML format check will fail, which can refer to [CI result](https://ci.pingcap.net/blue/organizations/jenkins/tikv_ghpr_test/detail/tikv_ghpr_test/25793/pipeline). I think we can add these two spaces, so that future poeple do not need to worry about the format.